docs/vectorizer/api-reference.md

```diff
@@ -145,6 +149,228 @@ in other management functions.
 
 The `int` id of the vectorizer that you created.
 
+## Loading configuration
+
+You use the loading configuration functions in `pgai` to define the way data is loaded from the source table.
+
+The loading functions are:
+
+- [ai.loading_column](#ailoading_column)
+- [ai.loading_uri](#ailoading_uri)
+
+### ai.loading_column
+
+You use `ai.loading_column` to load the data to embed directly from a column in the source table.
+
+#### Example usage
+
+```sql
+SELECT ai.create_vectorizer(
+    'my_table'::regclass,
+    loading => ai.loading_column('contents'),
+    -- other parameters...
+);
+```
+
+#### Parameters
+
+`ai.loading_column` takes the following parameters:
+
+| Name | Type | Default | Required | Description |
+|------|------|---------|----------|-------------|
+| column_name | text | - | ✔ | The name of the column containing the data to load. |
+
+#### Returns
+
+A JSON configuration object that you can use in [ai.create_vectorizer](#create-vectorizers).
+
+### ai.loading_uri
+
+You use `ai.loading_uri` to load the data to embed from a file that is referenced in a column of the source table.
+This file path is internally passed to [smart_open](https://github.com/piskvorky/smart_open), so it supports any protocol that smart_open supports, including:
+
+- Local files
+- Amazon S3
+- Google Cloud Storage
+- Azure Blob Storage
+- HTTP/HTTPS
+- SFTP
+- and [many more](https://github.com/piskvorky/smart_open/blob/master/help.txt)
+
+
+#### Environment configuration
+
+You just need to ensure the vectorizer worker has the correct credentials to access the file, such as in environment variables. Here is an example for AWS S3:
+
+```bash
+export AWS_ACCESS_KEY_ID='your_access_key'
+export AWS_SECRET_ACCESS_KEY='your_secret_key'
+export AWS_REGION='your_region'  # optional
+```
+
+Make sure these environment variables are properly set in the environment where the PGAI vectorizer worker runs.
+
+#### Example usage
+
+```sql
+SELECT ai.create_vectorizer(
+    'my_table'::regclass,
+    loading => ai.loading_uri('file_uri_column_name'),
+    -- other parameters...
+);
+```
+
+#### Parameters
+
+`ai.loading_uri` takes the following parameters:
+
+| Name | Type | Default | Required | Description |
+|------|------|---------|----------|-------------|
+| column_name | text | - | ✔ | The name of the column containing the file path. |
+
+#### Returns
+
+A JSON configuration object that you can use in [ai.create_vectorizer](#create-vectorizers).
+
+## Parsing configuration
+
+You use the parsing configuration functions in `pgai` to define how data is parsed after document loading. This is useful if for non-textual formats such as PDF documents.
+
+The parsing functions are:
+
+- [ai.parsing_auto](#aiparsing_auto): Automatically selects the appropriate parser based on file type.
+- [ai.parsing_none](#aiparsing_none): Converts various formats to Markdown.
```

Comment:
I think this is a left over mistake. Below we say that this option `skip the parsing step`